### DISCUSSION TITLE
MediaElch Video Tutorial

### DISCUSSION THREAD

**Discussion by GarfieldS** [Komet/MediaElch#1144] [General]
Just sharing as a resource. The best English speaking video on MediaElch is by xoxoHarls and can be found here. https://www.youtube.com/watch?v=KQ5hSNAY-qA

That is also saying something because even though it is 6 years old is still the best one that I have seen. I hope to make a video tutorial myself as some point.

**Comment by bugwelle** [comment]
Thanks! Back then MediaElch's icon didn't even have the headphones. :-)
